deploy/csi-blob-node.yaml

```diff
@@ -113,6 +115,10 @@ spec:
                   optional: true
             - name: CSI_ENDPOINT
               value: unix:///csi/csi.sock
+            - name: EDGE_CACHE_CONFIG_ENDPOINT
+              value: hydrad-config:8675
```

Comment:
same on parameters rather than hard coded

Reply:
There do not appear to be any params in the deploy tree. I'm not sure how this section of the tree is used currently, the addon pr appears to fork this yaml content.

I think we can do a param in helm, but not here.